### DISCUSSION TITLE
Programmatically setting track features for QuantitativeTrack

### DISCUSSION THREAD

**Discussion by mondegreen** [GMOD/jbrowse-components#3462] [Q&A]
I am trying to set up several JBrowse 2 instances each with several dozen tracks that are pre-configured for optimal viewing by researchers. There are several track configuration details that I cannot seem to set programmatically, such as the "height" of a track, or "Summary score mode", "histogram fill". While I can manually configure these after setting up the browser and saving the session, given the number of tracks I would like to do this programmatically so an optimal view is provided up front to users.

Can someone direct me to documentation about how I can do this or provide me with what options are available? I am coming from Jbrowse1 where all the data was stored locally and could be easily manipulated programmatically.

**Comment by cmdcolin** [comment]
Hi there,
The Summary score, and histogram fill are governed by settings on the renderer, it's a little tricky to access those settings since they are sort of 'sub-configs' of the track, but it looks like this

```json
{
  "type": "QuantitativeTrack",
  "trackId": "mytrack",
  "name": "mytrack",
  "assemblyNames": ["myasm"],
  "adapter": {
    "type": "BigWigAdapter",
    "bigWigLocation": { "uri": "file.bw"  }
  },
  "displays": [
    {
      "type": "LinearWiggleDisplay",
      "displayId": "mytrack-LinearWiggleDisplay",
      "renderers": {
        "XYPlotRenderer": {
          "type": "XYPlotRenderer",
          "filled": false,
          "summaryScoreMode": "max"
        }
      }
    }
  ]
}

```

The way I generated the above config was by taking a bigwig track, copying it with "Copy track" to a session track, then "Settings" on the session track, then going through the configuration to edit the particular rendering options that I wanted, and then going to "About track" dialog -> "Copy config" button

There is currently no setting for track height but this will be added by https://github.com/GMOD/jbrowse-components/issues/3030
